src/logic.py

```diff
@@ -54,11 +60,60 @@ def handle_queues(queue: Queue, processes: list):
                 dispatch_action_by_id(config.get("enable_gold_route_workflow_id"))
                 logger.debug(item['message'])
 
+            log_events(item)
+
         except Exception as ex:
             logger.error('Unknown error in logic. %s' % ex)
             traceback.print_exc(file=sys.stdout)
 
 
+def log_events(queu_item: object):
+    if ches_token_endpoint != "" and ches_username != "" and ches_password != "":
+        body = f"<p>The following change was detected by the switchover agent:</p> \
+        <p>{json.dumps(queu_item)}</p> \
+        <br> <p>{datetime.datetime.now()}</p>"
+
+        sendChesEmail(log_email, body)
+
+
+def sendChesEmail(to: str, body: str):
+    try:
+        namespace = config.get('namespace')
+        access_token = fetchChesToken()
+
+        if access_token is None:
+            logger.error('No Access Token Created')
```

Comment:
could return here, doesn't really matter but the sendemail will fail

Reply:
oh good catch I missed that one